daemon/daemon_windows.go

```diff
@@ -238,6 +240,29 @@ func checkSystem() error {
 		return fmt.Errorf("Failed to load vmcompute.dll. Ensure that the Containers role is installed.")
 	}
 
+	// Ensure that the required Host Network Service and vmcompute services
+	// are running. Docker will fail in unexpected ways if this is not present.
```

Comment:
Nit: Consider consistency in referencing the services in the comment, either `hns` and `vmcompute` or `Host Network Service` and `Hyper-V Host Compute Service`. Also, `s/if this is not present/if these are not present/`.